forte/processors/data_augment/algorithms/text_replacement_op.py

```diff
@@ -47,7 +47,7 @@ def replace(self, input_anno: Annotation) -> Tuple[bool, str]:
         returns the augmented string.
 
         Args:
-            input_anno: the input annotation to be replaced.
+            input_anno, the input annotation to be replaced.
```

Comment:
why is this changed to a comma?

Reply:
Thought I could fix another bug. I have changed it back.